keps/sig-node/4369-allow-special-characters-environment-variable/kep.yaml

```diff
@@ -29,6 +28,8 @@ latest-milestone: "v1.30"
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
   alpha: "v1.30"
+  beta: "v1.32"
+  stable: "v1.36"
```

Comment:
36 is rather pessimistic, don't you think :)

Reply:
If the goal is to soak the compatibility into all supported k8s versions, this might make sense?  I don't know if that was the thinking here..

Reply:
> If the goal is to soak the compatibility into all supported k8s versions, this might make sense

Yes, the intention is to have the beta version span four release cycle (the maximum range of version skew for our different components) to avoid compatibility issues.

Reply:
This is WAY more than we normally do for introducing new features

Reply:
For ratcheting, soak time can be counted starting at the initial alpha, because we always introduce "tolerate if read from storage" then.  And that's what matters when soaking.  Of if we go to stable one release after beta (v1.33), we still have a full support soak window.

Reply:
Thanks for the reminder, I will modify it to version v1.33 later.